.github/workflows/images.yml

```
@@ -68,3 +79,39 @@ jobs:
           publish_latest: ${{ steps.tags.outputs.tag_latest }}
           quay_user: ${{ secrets.QUAY_USER }}
           quay_password: ${{ secrets.QUAY_PASSWORD }}
+
+  update_argocd_dev:
+    name: Update ArgoCD in dev environment
+    # if: ${{ github.ref == 'refs/heads/main' }}
```

```suggestion
    if: ${{ github.ref == 'refs/heads/main' }}
```

commit this before merge